adkintunmobile-server/README.md

```diff
@@ -0,0 +1,15 @@
+# AdkintunMobile-Server Deploy
+Script and dockerfiles to deploy AdkintunMobile-Server.
+
+Previously run `run.sh` is important:
```

Comment:
I'd eliminate this line, because the following instructions are mandatory to run the script